sdks/python/apache_beam/runners/interactive/recording_manager_test.py

```diff
@@ -463,14 +463,13 @@ def test_clear(self):
     recording.wait_until_finish()
 
     # Assert that clearing only one recording clears that recording.
-    self.assertGreater(rm_1.describe()['size'], 0)
-    self.assertGreater(rm_2.describe()['size'], 0)
-    rm_1.clear()
-    self.assertEqual(rm_1.describe()['size'], 0)
-    self.assertGreater(rm_2.describe()['size'], 0)
-
-    rm_2.clear()
-    self.assertEqual(rm_2.describe()['size'], 0)
+    if rm_1.describe()['size'] > 0 and rm_2.describe()['size'] > 0:
```

Comment:
I don' t know this test very well, but it seems like that we expect both rm_1 and rm_2 to be greater than zero. If they're not greater than zero, that would indicate a problem, and the if condition just hides that problem.

Is there an alternate fix that ensures rm_1 and rm_2 are both greater than zero?

Reply:
You are right, I change it. As I can see, it's not really a problem but an expected behavior in some scenarios. There is actually a flag that indicates when the record stopped, and it can happen when it is not necessary to record the same data twice.
 I added another test for check single clear in one record and test this case apart.